integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/IdentityProvidersResource.java

```diff
@@ -43,6 +44,16 @@ public interface IdentityProvidersResource {
     @Produces(MediaType.APPLICATION_JSON)
     List<IdentityProviderRepresentation> findAll();
 
+    @GET
+    @Path("instances")
+    @Produces(MediaType.APPLICATION_JSON)
+    List<IdentityProviderRepresentation> find(@QueryParam("search") String search, @QueryParam("briefRepresentation") Boolean briefRepresentation, @QueryParam("first") Integer firstResult, @QueryParam("max") Integer maxResults);
+
+    @GET
+    @Path("instances/count")
+    @Produces(MediaType.APPLICATION_JSON)
+    Integer count(@QueryParam("search") String search);
```

Comment:
See my other comment for "count" endpoint

Reply:
Removed the count endpoint!